process/accounts/accountsProcessor.go

```diff
@@ -54,6 +56,7 @@ func NewAccountsProcessor(
 		internalMarshalizer:    marshalizer,
 		addressPubkeyConverter: addressPubkeyConverter,
 		balancePrecision:       math.Pow(10, float64(numDecimalsInFloatBalance)),
+		balancePrecisionESDT:   math.Pow(10, float64(core.MaxInt(numDecimalsInFloatBalanceESDT, 0))),
```

Comment:
no need for `core.MaxInt` => the constant is always higher than `0`.

Reply:
done.